app/views/curation_concerns/base/_form_files.html.erb

```diff
@@ -1,4 +1,27 @@
      <div id="fileupload">
+         <% if f.object.file_sets.present? > 0 %>
+             <fieldset id="uploaded-files">
+                 <legend>Previously Uploaded Files</legend>
+                 <table role="presentation" class="table table-striped uploaded-files">
+                     <tbody>
+                     <tr>
+                         <th>Filename</th>
+                         <th>Display Label</th>
+                     </tr>
```

Comment:
can you please switch this to `do ... end` instead of `{ }`. Our convention is to use the former for multi-line blocks.